Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -5806,6 +5830,8 @@ private void UpdateAchievements()
                             newAch2.Set(AchievementsReworked.WuffieCult);
                         if (!ach2.HasFlag(AchievementsReworked.DidYouGuardYourself) && player.PlayerRole == IRole.GuardianAngel && player.GAGuardWolfCount >= 3)
                             newAch2.Set(AchievementsReworked.DidYouGuardYourself);
+                        if (!ach2.HasFlag(AchievementsReworked.TheFireFighter) && player.PlayerRole == IRole.GuardianAngel && player.GACleanedHomeCount >= 2)
+                            newAch2.Set(AchievementsReworked.TheFireFighter);
```

Comment:
Can't the player get the achievement if they were guardian angel before but got stolen?
Also, why do we need to wait until the game end to give out the achievement?
I'd suggest giving it mid-game below line 4065.

And *if* the new achievement is given at the game end, it should go at the end of the list!